### PR TITLE
Rebrand to Alma Career Czechia in documentation and meta information

### PR DESCRIPTION
**Pull request by OndraM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Now you will be able to run the fix using `composer analyze` and execute automat
 On top of default code-style rules you are free to add any rules from [PHP-CS-Fixer] or [PHP_CodeSniffer].
 If needed, you can also override some default settings.
 
-Be aware you must add these settings **after** import of the base LMC code-style:
+Be aware you must add these settings **after** import of the base Alma Career (LMC) code-style:
```
Comment (jirinovak) [inline README.md:77]:
```suggestion
Be aware you must add these settings **after** import of the base Alma Career Czechia (LMC) code-style:
```

File: README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ See [EasyCodingStandard docs](https://github.com/symplify/easy-coding-standard#c
 
 You can configure your `ecs.php` to entirely skip some files, disable specific checks of suppress specific errors.
 
-Unlike adding/modifying checks, skips must be added **before** import of the base LMC code-style.
+Unlike adding/modifying checks, skips must be added **before** import of the base Alma Career (LMC) code-style.
```
Comment (jirinovak) [inline README.md:107]:
```suggestion
Unlike adding/modifying checks, skips must be added **before** import of the base Alma Career Czechia (LMC) code-style.
```

File: composer.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
     "name": "lmc/coding-standard",
-    "description": "Coding standard used in LMC projects",
+    "description": "Coding standard used in Alma Career Czechia (LMC) projects",
     "license": "MIT",
     "type": "library",
     "authors": [
         {
-            "name": "LMC s.r.o.",
-            "homepage": "https://github.com/lmc-eu"
+            "name": "Alma Career",
```
Comment (jirinovak) [inline composer.json:8]:
```suggestion
            "name": "Alma Career Czechia",
```

Comment (OndraM) [reply composer.json:8]: This refers to the name of the organization, which is Alma Career... I would keep this one as is (but I changed the README references as suggested).

**Review by jirinovak — APPROVED** [review]

**Pull request merged**